lib/src/code/text_style.dart

```diff
@@ -19,7 +19,7 @@ extension TextStyleExtension on TextStyle {
 
     return copyWith(
       color: Color.fromARGB(
-        clr.alpha ~/ 2,
+        127,
```

Comment:
Because we should avoid cases where we are pale already paled style

Reply:
This should not be a problem because `ancestorStyle` is never paled and retains the original opacity. It is only paled before the actual use in `TextStyle`. Have you actually seen this causing problems?

Reply:
You are right. Returned old logic